cdap-examples/FileSetExample/src/test/java/co/cask/cdap/examples/fileset/FileSetWordCountTest.java

```diff
@@ -56,7 +56,8 @@ public void testWordCountOnFileSet() throws Exception {
     final String line2 = "b a b";
 
     // discover the file set service
-    ServiceManager serviceManager = applicationManager.startService("FileSetService");
+    ServiceManager serviceManager = applicationManager.getServiceManager("FileSetService");
+    serviceManager.start();
```

Comment:
It depends on what they do. I would say at this point, we don't have to change all of them.